util/devel/test/vagrant/current/freebsd-FreeBSD-13.4-STABLE/Vagrantfile

```diff
@@ -6,13 +6,18 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 Vagrant.configure("2") do |config|
-  config.vm.box = "freebsd/FreeBSD-13.4-STABLE"
+  config.vm.box = "bento/freebsd-13.4"
```

Comment:
We could consider renaming the dir this file is in. It could be `bento-freebsd-13.4`. I don't recall if any tooling would care about that change (the main thing to check is the script to generate the commands to install dependencies).

Not a big deal and leaving it alone is fine too.